src/client/components/veteran-information/BirthInformationSection.jsx

```diff
@@ -55,6 +55,7 @@ class BirthInformationSection extends React.Component {
               name="cityOfBirth"
               field={this.props.data.cityOfBirth}
               autocomplete="off"
+              charMax={20}
```

Comment:
Just want to verify that this has to be 20 max characters, but city in `Address` can be 30?